NuGet.Docs/create/Versioning.cshtml

```diff
@@ -0,0 +1 @@
+@{Response.RedirectPermanent("~/ndocs/create-packages/dependency-versions");}
```

Comment:
Is `ndocs` a valid target? (more of these in the PR, not commenting on all of them)